pyproject.toml

```diff
@@ -26,3 +26,15 @@ Issues = "https://github.com/AbanteAI/rawdog/issues"
 
 [project.scripts]
 rawdog = "rawdog.__main__:main"
+
+[tool.ruff]
+line-length = 120
+extend-exclude = ["examples", "src/rawdog/prompts.py"]
```

Comment:
ignoring `prompts.py` in particular due to many prompt lines being over 120 characters long